include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -702,49 +702,40 @@ __parallel_radix_sort(_ExecutionPolicy&& __exec, _Range&& __in_rng)
-        using _RadixBitsType = ::std::integral_constant<::std::uint32_t, __radix_bits>;
-        using _AscendingType = ::std::bool_constant<__is_ascending>;
-        using _CustomName = typename __decay_t<_ExecutionPolicy>::kernel_name;
-        using _RadixSortKernel = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_generator<
-            __radix_sort_one_group, _CustomName, _RadixBitsType, _AscendingType, __decay_t<_Range>>;
-
-        if (__n <= 64)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 9>, __wg_size, 1, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 128)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 8>, __wg_size * 2, 1, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 256)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 0>, __wg_size * 2, 2, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 512)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 1>, __wg_size * 2, 4, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 1024)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 2>, __wg_size * 2, 8, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 2048)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 3>, __wg_size * 4, 8, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 4096)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 4>, __wg_size * 4, 16, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 8192)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 5>, __wg_size * 8, 16, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else if (__n <= 16384)
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 6>, __wg_size * 8, 32, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        else
-        {
-            assert(__n <= 32768);
-            __event = __subgroup_radix_sort<__i_kernel_name<_RadixSortKernel, 7>, __wg_size * 8, 64, __radix_bits,
-                                            __is_ascending>{}(__exec.queue(), __in_rng);
-        }
-    }
+    using _RadixSortKernel = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_generator<
```

Comment:
It looks like some lines of code were lost here `_CustomName`, `_RadixBitsType`, `_AscendingType` are all undefined as far as I can tell.

Reply:
oops, it seems is "resolving conflict artefact"... Thank you, Dan.